plugins/package-managers/gradle/src/funTest/kotlin/GradleAndroidFunTest.kt

```diff
@@ -33,7 +33,8 @@ class GradleAndroidFunTest : StringSpec({
         val definitionFile = getAssetFile("projects/synthetic/gradle-android/build.gradle")
         val expectedResultFile = getAssetFile("projects/synthetic/gradle-android-expected-output-root.yml")
 
-        val result = create("Gradle").resolveSingleProject(definitionFile, resolveScopes = true)
+        val result = create("Gradle", "javaVersion" to "17")
```

Comment:
Should we factor out `createGradle()` so these can be kept one-liners ?

Reply:
I have a slight preference for seeing the configuration explicitly in each test, as Java requirements might change on a per-test basis. Also, I'd like to keep this aligned with the respective GI tests as much as possible for now.